docs/compilers/CSharp/Compiler Breaking Changes - DotNet 10.md

```diff
@@ -1,4 +1,6 @@
-# This document lists known breaking changes in Roslyn after .NET 9 all the way to .NET 10.
+# Roslyn breaking changes from .NET 9.0.100 through .NET 10.0.100
+
+This document lists known breaking changes in Roslyn after .NET 9 general release (.NET SDK version 9.0.100) through .NET 10 general release (.NET SDK version 10.0.100).
```

Comment:
As a side note, Bill, I don't remember whether we'd decided on chronological or reverse chronological order for entries in each document. We've been appending at the end (ie. chronological). Does that match your expectation?

Reply:
That works. We've had no concerns on the order.